<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormularioObjeto</class>
 <widget class="QWidget" name="FormularioObjeto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FormularioObjeto {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #f8f9fa, stop: 1 #e9ecef);
    border: none;
    color: #212529;
}

QLabel {
    font: 600 12pt "Segoe UI";
    color: #495057;
    background: transparent;
    padding: 5px;
}

QLineEdit {
    background-color: #ffffff;
    border: 2px solid #dee2e6;
    border-radius: 8px;
    padding: 10px 12px;
    font: 11pt "Segoe UI";
    color: #212529;
    selection-background-color: #e3f2fd;
    selection-color: #1976d2;
}

QLineEdit:focus {
    border-color: #80bdff;
    box-shadow: 0 0 0 3px rgba(0, 123, 255, 0.25);
}

QTextEdit {
    background-color: #ffffff;
    border: 2px solid #dee2e6;
    border-radius: 8px;
    padding: 10px 12px;
    font: 11pt "Segoe UI";
    color: #212529;
    selection-background-color: #e3f2fd;
    selection-color: #1976d2;
}

QTextEdit:focus {
    border-color: #80bdff;
    box-shadow: 0 0 0 3px rgba(0, 123, 255, 0.25);
}

QSpinBox {
    background-color: #ffffff;
    border: 2px solid #dee2e6;
    border-radius: 8px;
    padding: 8px 12px;
    font: 11pt "Segoe UI";
    color: #212529;
    min-height: 25px;
}

QSpinBox:focus {
    border-color: #80bdff;
    box-shadow: 0 0 0 2px rgba(0, 123, 255, 0.25);
}

QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #007bff, stop: 1 #0056b3);
    color: white;
    border: none;
    border-radius: 10px;
    font: bold 12pt "Segoe UI";
    padding: 12px 20px;
    min-width: 100px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #0056b3, stop: 1 #004085);
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #004085, stop: 1 #002752);
}

QPushButton#pushButtonLimpiar {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #6c757d, stop: 1 #495057);
}

QPushButton#pushButtonLimpiar:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #5a6268, stop: 1 #343a40);
}

QPushButton#pushButtonLimpiar:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #495057, stop: 1 #212529);
}</string>
  </property>
  <property name="windowTitle">
   <string>Publicar Objeto</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="styleSheet">
      <string notr="true">QLabel {
    font: 700 20pt "Segoe UI";
    color: #2c3e50;
    background: transparent;
    padding: 10px;
    border-radius: 8px;
}</string>
     </property>
     <property name="text">
      <string>Publicar Nuevo Objeto</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetContenedor" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widgetContenedor {
    background-color: rgba(255, 255, 255, 250);
    border: 2px solid #dee2e6;
    border-radius: 15px;
    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutContenedor">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldGrowthPolicy::ExpandingFieldsGrow</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::RowWrapPolicy::DontWrapRows</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>25</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Nombre del objeto:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditNombre">
       <property name="placeholderText">
        <string>Ej: Taladro eléctrico, Libro de cocina...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Categoría:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditCategoria">
       <property name="placeholderText">
        <string>Ej: Herramientas, Libros, Electrodomésticos...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tiempo disponible:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_tiempo">
       <item>
        <widget class="QSpinBox" name="spinBoxTiempo">
         <property name="suffix">
          <string> días</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>365</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Descripción:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="textEditDescripcion">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Describe el objeto, su estado, cuidados especiales, etc.</string>
       </property>
      </widget>
     </item>
    </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_botones">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLimpiar">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGuardar">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>Guardar</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
